input/cql/IntravesicalBacillusCalmetteGuerinForBladderCancerFHIR.cql

```diff
@@ -92,6 +99,9 @@ define "Cystectomy Done":
       such that Cystectomy.performed.toInterval ( ) 6 months or less before start of FirstBladderCancerStaging.performed.toInterval ( )
     where Cystectomy.status = 'completed'
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/ 
```

Comment:
As written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis

Reply:
Updated comment to be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis.